### PR TITLE
Replace AE help with Annotare help links

### PR DESCRIPTION
**Pull request by anjaf**
This is for making Annotare a step more ready for ArrayExpress homepage to retire. 
I've updated links from within Annotare or the submission help pages pointing to ArrayExpress help to point to Annotare help. 

There are still some links pointing to "how to search in AE" and "how to find array designs". These need to say active for now until BioStudies has been tested and is ready to take over.

### CODE REVIEW

File: app/om/src/main/java/uk/ac/ebi/fg/annotare2/submission/model/FileType.java [hunk 1]
```diff
@@ -21,11 +21,11 @@
...
+    PROCESSED_FILE("Processed", "A tab-delimited text (.txt) file containing data from a single assay <br/><a href=\"https://www.ebi.ac.uk/fg/annotare/help/accepted_processed_ma_file_formats.html\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted microarray processed file formats</a><br/><a href=\"https://wwwdev.ebi.ac.uk/fg/annotare/help/accepted_sequencing_file_formats.html#processed_data\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted sequencing processed file formats</a><div> <img src=\"../../assets/images/graphic_processed.png\" alt=\"One Color\" width=\"250\" height=\"125\"></img> </div> "),
```
Comment (haideriqbal) [inline app/om/src/main/java/uk/ac/ebi/fg/annotare2/submission/model/FileType.java:28]: Can we please change this link from pointing to `wwwdev` to `www`

**Review by haideriqbal — APPROVED** [review]
All good! Just address one minor comment before merging the branch!

File: app/om/src/main/java/uk/ac/ebi/fg/annotare2/submission/model/FileType.java (hunk 1 above)
Comment (anjaf) [reply app/om/src/main/java/uk/ac/ebi/fg/annotare2/submission/model/FileType.java:28]: Good catch. Copy/paste error...
Can we use relative links from this page? Or is it better to keep the full URL?

Comment (haideriqbal) [reply app/om/src/main/java/uk/ac/ebi/fg/annotare2/submission/model/FileType.java:28]: We can use relative links as well :) Usually using relative links is generally considered best practice :)

**Comment by anjaf** [comment]
@haideriqbal I've fixed the erroneous link to wwwdev. I wasn't sure how the relative link would work from within the Java code, so left this for now.

**Comment by haideriqbal** [comment]
Looks good @anjaf . Just for reference, they work the same way as in the normal HTML page. For example, if you see how the image has been referenced in the file you just fixed `../../assets/images/graphic_processed.png` in the same way you can reference help pages as well :)

**Pull request merged**